README.md

```diff
@@ -19,20 +19,21 @@ Your contributions and suggestions are heartily♥ welcome. (✿◕‿◕). Plea
   * [Operating Systems](#operating-systems)
   * [Penetration Testing Report Templates](#penetration-testing-report-templates)
   * [Code examples for Penetration Testing](#code-examples-for-penetration-testing)
+* [Network vulnerability scanners](#network-vulnerability-scanners)
```

Comment:
I like that you've broken out some of these subsections from the unwieldly "Tools" section in this list. I think that's the right move. Moving forward, I think we should definitely deprecate the "Tools" heading, since almost everything on this list can be broadly categorized in that way.

I'd like to see the other headings move to more appropriate places as well, but I don't think this PR needs to handle that on its own if you want to focus on making this Awesome List compliant first.